### DISCUSSION TITLE
More about İbax !!!!!

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#1328] [Ideas]
There is a lot of talk about Ibax, but none of them is definitive information.
Can an official or someone else give us precise information and some confidence?
I don't like doing business with this uncertainty.  
Thank you from now....

![samsung_galaxy_fold_blue_butterfly_4k-1600x900](https://user-images.githubusercontent.com/98268472/151663966-fb540376-26a4-422c-a718-7926c38941db.jpg)

**Comment by ghost** [comment]
+1
